test/validation/api/atomic/atomic.c

```diff
@@ -583,13 +718,21 @@ static void test_atomic_store(void)
 	odp_atomic_store_u64(&global_mem->a64u_max, U64_INIT_VAL);
 	odp_atomic_store_u32(&global_mem->a32u_xchg, U32_INIT_VAL);
 	odp_atomic_store_u64(&global_mem->a64u_xchg, U64_INIT_VAL);
+
+	/* Set global odp_u128_t variable to U64_INIT_VAL before calling
+	 * test_atomic_store(). */
+	test_atomic_init_u128(&a128u_tmp, U64_INIT_VAL);
+	odp_atomic_store_u128(&global_mem->a128u, a128u_tmp);
 }
 
 static void test_atomic_validate(int check)
 {
 	CU_ASSERT(U32_INIT_VAL == odp_atomic_load_u32(&global_mem->a32u));
 	CU_ASSERT(U64_INIT_VAL == odp_atomic_load_u64(&global_mem->a64u));
 
+	CU_ASSERT(U64_INIT_VAL == global_mem->a128u.v.u64[0]);
+	CU_ASSERT(U64_INIT_VAL == global_mem->a128u.v.u64[1]);
```

Comment:
This is something that I wanted to discuss as well. I had originally used the odp_atomic_load_u128() for assertion checks and written something as below:
```
odp_u128_t a128u_tmp;
a128u_tmp.u64[0] = U64_INIT_VAL;
a128u_tmp.u64[1] = U64_INIT_VAL;
CU_ASSERT(a128u_tmp == odp_atomic_load_u128(&global_mem->a128u));
```
But the build fails with an error like below:
```
atomic.c:725:22: error: invalid operands to binary == (have ‘odp_u128_t’ {aka ‘struct odp_u128_s’} and ‘odp_u128_t’ {aka ‘struct odp_u128_s’})
  725 |  CU_ASSERT(a128u_tmp == odp_atomic_load_u128(&global_mem->a128u));
      |                      ^~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                         |
      |                         odp_u128_t {aka struct odp_u128_s}

```
Since there is no thread contention as only the main thread calls `test_atomic_validate()`, so  I changed it to test each 64-bit unit. If you have any suggestions on how to resolve the issue, I would really appreciate it.